frontend/src/components/Checkbox/Checkbox.types.ts

```diff
@@ -0,0 +1,10 @@
+interface CheckboxProps {
+  label?: string;
+
+  checked: boolean;
+
+  onChange?: (checked: boolean) => void;
```

Comment:
As I see checked is required, bit onChange is not, so if there is no handler form parent component, `checked` is never gonna change. Not sure thats expected behavior